pygeoapi_config_dialog_base.ui

```diff
@@ -58,9 +58,6 @@
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="1">
            <widget class="QLineEdit" name="lineEditHost">
-            <property name="inputMask">
```

Comment:
I think I did not explain myself well. The host will always be an [IP address](https://github.com/geopython/pygeoapi/blob/6158ed54a9a8dfc4e3e4b979e6551355033e0d40/pygeoapi/schemas/config/pygeoapi-config-0.x.yml#L18), so you can leave this mask on. It is the url, that can be an environmental variable.